src/cider/exec/nextgen/context/CodegenContext.h

```diff
@@ -225,6 +224,14 @@ class CodegenContext {
   // returns an index used for retrieving the charset at runtime
   int registerTrimStringOperCharMap(const std::string& trim_chars);
 
+ private:
```

Comment:
why needs to move here out of previous private section?

Reply:
Just to make code in good order.